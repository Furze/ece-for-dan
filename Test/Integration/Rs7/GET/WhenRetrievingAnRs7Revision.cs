using Bard;
using MoE.ECE.Domain.Read.Model.Rs7;
using MoE.ECE.Integration.Tests.Chapter;
using MoE.ECE.Integration.Tests.Infrastructure;
using Shouldly;
using Xunit;
using Xunit.Abstractions;

namespace MoE.ECE.Integration.Tests.Rs7.GET
{
    public class WhenRetrievingAnRs7Revision : SpeedyIntegrationTestBase
    {
        public WhenRetrievingAnRs7Revision(RunOnceBeforeAllTests testSetUp, ITestOutputHelper output,
            TestState<ECEStoryBook, ECEStoryData> testState) : base(testSetUp, output, testState)
        {
        }

        private const string Url = "api/rs7";

        protected override void Arrange()
        {
            Given
                .A_rs7_skeleton_has_been_created(setup => setup.FundingPeriodYear = 2020)
                .An_rs7_is_ready_for_internal_ministry_review()
                .GetResult(storyData => Rs7Model = storyData.Rs7Model);
        }

        protected override void Act()
        {
            When.Get($"{Url}/{Rs7Model.Id.ToString()}?revisionNumber=1");
        }

        private Rs7Model Rs7Model
        {
            get => TestData.Rs7Model;
            set => TestData.Rs7Model = value;
        }

        [Fact]
        public void ThenTheBusinessEntityIdShouldBePopulated()
        {
            Then
                .Response
                .Content<Rs7Model>()
                .BusinessEntityId
                .ShouldNotBeNull();
        }

        [Fact]
        public void ThenTheOrganisationIdShouldBePopulated()
        {
            Then
                .Response
                .Content<Rs7Model>()
                .OrganisationId
                .ShouldNotBeNull();
        }

        [Fact]
        public void ThenTheResponseShouldBeOk()
        {
            Then
                .Response
                .ShouldBe
                .Ok<Rs7Model>();
        }

        [Fact]
        public void ThenTheRevisionShouldBePopulated()
        {
            var rs7AndRevision = Then.Response
                .Content<Rs7Model>();

            rs7AndRevision.Id.ShouldBeGreaterThan(0); // Should be Rs7.Id, not Rs7Revision.Id

            rs7AndRevision.RevisionNumber.ShouldBe(1);

            rs7AndRevision.AdvanceMonths.ShouldNotBeNull();
            rs7AndRevision.AdvanceMonths?.Length.ShouldBe(4);

            rs7AndRevision.EntitlementMonths.ShouldNotBeNull();
            rs7AndRevision.EntitlementMonths?.Length.ShouldBe(4);

            rs7AndRevision.IsAttested.ShouldNotBeNull();
        }
        
        [Fact]
        public void Then_the_response_snapshot_should_be_ok()
        {
            Then.Snapshot().Match<Rs7Model>(IgnoreFieldsFor.Rs7Model);
        }
    }
}