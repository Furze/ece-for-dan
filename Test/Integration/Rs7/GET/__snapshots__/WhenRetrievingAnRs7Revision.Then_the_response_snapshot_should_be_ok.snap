{
  "Id": 1,
  "BusinessEntityId": "0f3d22c5-818d-40d7-85f7-b58573fd9bc2",
  "OrganisationId": 114895,
  "FundingPeriod": "July",
  "FundingYear": 2021,
  "FundingPeriodYear": 2020,
  "IsZeroReturn": false,
  "RollStatus": "InternalReadyForReview",
  "ReceivedDate": "2020-11-23T14:27:22.2890219+13:00",
  "RevisionNumber": 1,
  "RevisionDate": "2020-11-23T14:27:22.3611026+13:00",
  "AdvanceMonths": [
    {
      "MonthNumber": 7,
      "Year": 2020,
      "AllDay": 2,
      "Sessional": null,
      "ParentLed": null
    },
    {
      "MonthNumber": 8,
      "Year": 2020,
      "AllDay": 2,
      "Sessional": null,
      "ParentLed": null
    },
    {
      "MonthNumber": 9,
      "Year": 2020,
      "AllDay": 2,
      "Sessional": null,
      "ParentLed": null
    },
    {
      "MonthNumber": 10,
      "Year": 2020,
      "AllDay": 2,
      "Sessional": null,
      "ParentLed": null
    }
  ],
  "EntitlementMonths": [
    {
      "MonthNumber": 2,
      "Year": 2020,
      "Days": [
        {
          "DayNumber": 1,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 2,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 3,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 4,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 5,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 6,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 7,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 8,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 9,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 10,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 11,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 12,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 13,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 14,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 15,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 16,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 17,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 18,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 19,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 20,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 21,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 22,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 23,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 24,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 25,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 26,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 27,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 28,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 29,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        }
      ]
    },
    {
      "MonthNumber": 3,
      "Year": 2020,
      "Days": [
        {
          "DayNumber": 1,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 2,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 3,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 4,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 5,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 6,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 7,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 8,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 9,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 10,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 11,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 12,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 13,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 14,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 15,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 16,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 17,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 18,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 19,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 20,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 21,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 22,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 23,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 24,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 25,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 26,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 27,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 28,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 29,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 30,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 31,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        }
      ]
    },
    {
      "MonthNumber": 4,
      "Year": 2020,
      "Days": [
        {
          "DayNumber": 1,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 2,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 3,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 4,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 5,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 6,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 7,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 8,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 9,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 10,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 11,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 12,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 13,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 14,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 15,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 16,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 17,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 18,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 19,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 20,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 21,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 22,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 23,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 24,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 25,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 26,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 27,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 28,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 29,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 30,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        }
      ]
    },
    {
      "MonthNumber": 5,
      "Year": 2020,
      "Days": [
        {
          "DayNumber": 1,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 2,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 3,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 4,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 5,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 6,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 7,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 8,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 9,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 10,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 11,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 12,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 13,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 14,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 15,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 16,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 17,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 18,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 19,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 20,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 21,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 22,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 23,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 24,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 25,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 26,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 27,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 28,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 29,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 30,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        },
        {
          "DayNumber": 31,
          "Under2": 18,
          "TwoAndOver": 1,
          "Hours20": null,
          "Plus10": 23,
          "Certificated": 1,
          "NonCertificated": 2
        }
      ]
    }
  ],
  "IsAttested": true,
  "Declaration": {
    "FullName": "Donald Trump",
    "ContactPhone": "021509753",
    "Role": "President",
    "IsDeclaredTrue": true
  },
  "RequestId": "76087"
}
